gpkit/breakdown.py

```diff
@@ -0,0 +1,92 @@
+"""Define the Breakdown class"""
+import numpy.testing as npt
+from gpkit import Model, Variable
+
+
+class Breakdown(Model):
+    def __init__(self, input_dict):
+        """
+        model to make graphical breakdowns of weights, etc.
+        takes input as a dict, variable arguments provided in a list
+        """
+        #create the list of variables to make constraints out of
+        self.constr = []
+        self.varlist = []
+        #call recursive function to create gp constraints
+        total = self.recurse(input_dict)
+
+        self.sol = self.solve_method()
+
+    def getSolution(self):
+        return self.sol
+
+    def solve_method(self):
+        m = Model(self.make_objective(), self.make_constraints())
+        return m.solve(verbosity=0)
```

Comment:
I don't think this method is needed. no need to solve the model on initialization -- let a user cal `solve()` instead.